src/App.vue

```diff
@@ -20,6 +20,8 @@
 <script lang="ts">
 import Vue from 'vue'
 import FilterContainer from './components/FilterContainer.vue'
+import CheckboxFilter from './components/CheckboxFilter.vue'
+import StringFilter from './components/StringFilter.vue'
```

Comment:
Suggest to collapse filter imports to single line
`import { FilterContainer, StringFilter, CheckboxFilter } from './components'`